style.css

```
@@ -1,14 +1,42 @@
 body {
-    background-color: wheat;
-    font-family: Helvetica, sans-serif;
+    background-color: beige;
+    font-family: 'Allura'; font-size: 35px;
```

one property per line
```suggestion
    font-family: 'Allura'; 
    font-size: 35px;
```